bips/obpp-03.mediawiki

```diff
@@ -71,10 +73,27 @@ When one of the following cases exist, a compliant implementation may create an
 ## Add a spend output to the transaction.
 ## Add a change output to the transaction containing the difference between the current set of inputs and the desired spend.
 # Repeat step 1 to create a second spend output and change output.
+# Repeat step 2 zero or more times.
 # Adjust the change outputs as necessary to pay the desired transaction fee.
 
 Clients which create intentional HITs must have the capability to form alternate form HITs, and must do so for a non-zero fraction of the transactions they create.
 
+===Rules===
+
+A HIT formed via the proceding procedure will adhere to the following conditions:
+
+# The number of unique inputs scripts must exceed the number of output scripts.
+# All output scripts must be unique.
+# At least one pair of outputs must be of equal value.
+## "Standard outputs" refers to the set of outputs with equal value
+## "Standard value" refers to the value of the standard outputs
+## "Change outputs" refers to all outputs which are not standard outputs
+# For a HIT containing n standard outputs, there must exist at least one possible way to organize the inputs and outputs into n sets, where all sets satisfy the following:
```

Comment:
The transaction will have n standard outputs and n change outputs for a total of 2n outputs per HIT.